modules/express/src/clientRoutes.ts

```diff
@@ -1088,6 +1114,12 @@ export function setupAPIRoutes(app: express.Application, config: Config): void {
   // send transaction
   app.post('/api/v2/:coin/wallet/:id/sendcoins', parseBody, prepareBitGo(config), promiseWrapper(handleV2SendOne));
   app.post('/api/v2/:coin/wallet/:id/sendmany', parseBody, prepareBitGo(config), promiseWrapper(handleV2SendMany));
+  app.post(
+    '/api/v2/:coin/wallet/:id/prebuildAndSignTx',
+    parseBody,
+    prepareBitGo(config),
+    promiseWrapper(handlePrebuildAndSignTx)
+  );
```

Comment:
Can we stay consistent with the handler names of the other routes. Something like 
```suggestion
    '/api/v2/:coin/wallet/:id/prebuildAndSignTransaction',
    parseBody,
    prepareBitGo(config),
    promiseWrapper(handleV2PrebuildAndSignTransaction)
  );
```